src/main/java/frc/robot/Robot.java

```diff
@@ -30,10 +33,8 @@
  */
 public class Robot extends LoggedRobot
 {
-	private static final String defaultAuto = "Default";
-	private static final String customAuto = "My Auto";
-	private String autoSelected;
-	private final LoggedDashboardChooser<String> chooser = new LoggedDashboardChooser<>("Auto Choices");
+	private Command autonomousCommand;
+	private CrabbyContainer robotContainer;
```

Comment:
This is the incorrect way to init container. Don't worry about doing that. I have pr open. Please review #29